inference-runtime/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2021 Intel Corporation
+
+cmake_minimum_required(VERSION 3.20)
+
+set(LIBTORCH_URL "https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.11.0%2Bcpu.zip")
+set(LIBPADDLE_URL "https://paddle-inference-lib.bj.bcebos.com/2.3.0/cxx_c/Linux/CPU/gcc5.4_openblas/paddle_inference.tgz")
+set(LIBTENSORFLOW_URL "https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-cpu-linux-x86_64-2.8.0.tar.gz")
+set(LIBONNXRUNTIME_URL "https://github.com/microsoft/onnxruntime/releases/download/v1.7.0/onnxruntime-linux-x64-1.7.0.tgz")
```

Comment:
Onnx has upgraded to version 1.11.0 in CCAI1.3 release.  
https://github.com/microsoft/onnxruntime/releases/download/v1.11.0/onnxruntime-linux-x64-1.11.0.tgz

Reply:
updated